.circleci/config.yml

```diff
@@ -58,59 +48,48 @@ apple_defaults: &apple_defaults
 jobs:
   android:
     executor:
-      name: android/android
-      sdk-version: '28'
-      variant: ndk
+      name: android/android-docker
+      tag: 2022.04.1-ndk
+    working_directory: /home/circleci
     environment:
-      - HERMES_WS_DIR: /tmp/hermes
-      - TERM: dumb
-      - DEBIAN_FRONTEND: noninteractive
+      HERMES_WS_DIR: /home/circleci
     steps:
-      - checkout
       - run:
           name: Set up workspace and install dependencies
           command: |
             yes | sdkmanager "cmake;3.18.1" &
```

Comment:
Maybe use https://github.com/react-native-community/docker-android, it has all the deps bundled.